### ISSUE TITLE
Import Records from external file

### ISSUE THREAD

**Issue by feliciabeaudry** [subjectsplus/SubjectsPlus#1511]
I think it would be very helpful if users could import Records from an external file.  For libraries that have a large number of electronic resources, they may have a list of them (or access to one through a report) already in a CSV file or spreadsheet.  If output from such a file could be uploaded for the purposes of creating Records, it would be an incredible time saver, especially for new users.

**Comment by sandbergja** [comment]
Agreed that this would be nice.  In case it helps, here is the code that we used to migrate our data from our old LibGuides 2 site (which exports records in an XML format): https://github.com/subjectsplus/SubjectsPlus/pull/1372/files
